packages/unminify/src/transformations/__tests__/un-jsx.spec.ts

```diff
@@ -172,16 +171,15 @@ inlineTest('jsx with dynamic Component tag #2',
   `
 function fn() {
   return React.createElement(
-    components[0],
+    Components,
```

Comment:
Same here.

Reply:
previously it was working like 
input 
```
React.createElement(components[0], null, "Hello");
```
output 
```
const Component = components[0];
<Component>Hello</Component>; 
```

It was creating a variable as a tag and look like this was not picking components[0] also there were not intialized components variable.


According to this PR flow, I can update this, please check below example 

input

```
const Components = ['div']
React.createElement(
    Components[0],
    null,
    "Hello",
  );
```

output 

```
const Components = ['div'];
<div>Hello</div>
```

Reply:
Oh, then it's ok. My suggestion is that we should always keep existing test cases. We can update the output according to the changes and add some new test cases to demonstrate the new behavior.

---

So we are going to do some evaluation things to determine if we can inline the tag?

Reply:
Noted. Yep, I will update the MR description, as I have handled a few scenarios with this MR and the ticket only mentions one. And let me update the code for the above test case.